### ISSUE TITLE
Enable null safety for dart:ui

### ISSUE THREAD

**Issue by mit-mit** [flutter/flutter#53661]
Tracking issue for enabling null safety (https://github.com/dart-lang/language/issues/110) in `dart:ui`

**Comment by pcsosinski** [comment]
@yjbanov @cbracken is this work underway?

**Comment by franklinyow** [comment]
How are we doing on this?

**Comment by yjbanov** [comment]
The PRs referenced above should provide 100% coverage for `dart:ui` using temporary annotations. Once those land, I'll follow-up with a PR that will apply the new syntax. After that, I'll move on to `dart:_engine`, but that can happen in parallel with framework migration.

**Comment by pcsosinski** [comment]
@a14n as FYI

thanks Yegor!

**Comment by yjbanov** [comment]
One of the semantics.dart PRs was rolled back. The new PR is https://github.com/flutter/engine/pull/18791. Assuming all 5 PRs land and there are no new `dart:ui` API additions in the meantime, we'll have 100% coverage.

**Comment by yjbanov** [comment]
This is fixed now.
